requirements.txt

```diff
@@ -1,6 +1,6 @@
 pytest~=6.2.5
 flake8~=4.0.1
 mypy==1.5.1
-git+https://github.com/numpy/numpy-stubs
+numpy~=1.20
```

Comment:
Are we OK with just allowing every numpy version since 1.20 or do we want to pin to a specific minor version?

Reply:
I noticed, that `cadquery` requires at least `numpy 1.22` but doesn't specify it...